test/libsolidity/ABIDecoderTests.cpp

```diff
@@ -663,18 +663,18 @@ BOOST_AUTO_TEST_CASE(empty_struct)
 {
```

Comment:
The test case can be removed completely, it tests the semantics of decoding an empty struct, which is not possible anymore.